doc/ref_arch/openstack/chapters/chapter08.md

```diff
@@ -19,6 +19,8 @@ The most obvious way to address the gaps will be to propose a set of APIs in the
 
 * Public and Multi-Cloud  - The framework offered by OpenStack does not support multi-cloud deployments, along with the core deployments
 
+From the short term point of view, we could use ONAP to deal with this gap. However, for a long term point of view, reliance on ONAP should be avoided and the OpenStack APIs should be implemented/enhanced to address this gap.
```

Comment:
In the short-term, how does ONAP support multi-cloud deployments? Is it a central scheduler that decides which cloud the deployment should be made on? Is it capable of creating a multi-cloud overlay, if that is one of the needs of a multi-cloud deployment? Etc. etc.

In the long-term, what is the nature of the APIs that we are requesting for which OSTK services beyond Identity (Keystone) and Dashboard (Horizon).

Reply:
@sukhdevkapur Can ONAP be replaced by "an orchestrator such as ONAP" to make it more open?